test/index.sh

```diff
@@ -76,6 +76,7 @@ test 'TESTCASE: --retries and --bail should work well together' test/bail-and-re
 test 'TESTCASE: subprocess exits before sending an end message' test/no-subprocess-end/index.js
 test 'TESTCASE: unhandled rejections should not force subprocess to exit' test/q-promises/index.js
 test 'TESTCASE: uncaught exceptions should not force subprocess to exit' test/uncaught-exception/index.js
+test 'TESTCASE: grep option' test/grep/index.js
```

Comment:
it would also be great to have another test which checks the programmatic usage of `grep()` method, however if you feel stuck with the code above, disregard my request here and I will do it on my own.

Reply:
That's nice. Here's the example: https://github.com/qatechniker/mocha-parallel-tests/tree/773125c2f1a387d660b101fd7c37ef9f4c255fc8/test/run-programmatically/reporter-done
Please don't put the new test case into the same directory (test/run-programmatically), just put in in "test" directory instead.